src/main/scala/io/iohk/ethereum/vm/VM.scala

```diff
@@ -13,9 +14,9 @@ object VM {
     * @param context context to be executed
     * @return result of the execution
     */
-  def run(context: ProgramContext): ProgramResult = {
-    val finalState = run(ProgramState(context))
-    ProgramResult(finalState.returnData, finalState.storage, finalState.error)
+  def run(env: ExecEnv, storage: Storage): ProgramResult = {
```

Comment:
`ExecEnv` stores `val`s which are not supposed to be changed during execution of a program. It's an implementation of vals listen in section 9.3. of YP _"Execution environment"_.
I updated docs.